flink/v1.12/flink/src/main/java/com/netease/arctic/flink/write/ArcticFileWriter.java

```diff
@@ -148,16 +148,7 @@ private void initTaskWriterFactory(Long mask) {
   }
 
   private Long getTransactionId() {
```

Comment:
After removing transaction in ArcticFileWriter.java, it's useless to initTaskWriterFactory(null) in processElement. And the input parameter of mask can be replaced with long in initTaskWriterFactory().

Reply:
I removed it.